examples/vm_scheduling/reinforcement_learning/common/components/state.py

```diff
@@ -0,0 +1,121 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import numpy as np
+import pandas as pd
+
+
+PM_ATTRIBUTES = ["cpu_cores_capacity", "memory_capacity", \
+                 "cpu_cores_allocated", "memory_allocated"]
+
+VM_ATTRIBUTES = ["cpu_cores_requirement", "memory_requirement", "lifetime", "remain_time", "total_income"]
```

Comment:
Since the latter 3 are actual information got by peeking. 
How about adding switches in the config to decide to use them or not?

Reply:
I'll add a switch